x-pack/plugins/transform/public/alerting/transform_health_rule_type/register_transform_health_rule.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { lazy } from 'react';
+import { i18n } from '@kbn/i18n';
+import { TRANSFORM_RULE_TYPE } from '../../../common';
+import type { TransformHealthRuleParams } from '../../../common/types/alerting';
+import type { AlertTypeModel } from '../../../../triggers_actions_ui/public';
+
+export function createTransformHealthRuleType(): AlertTypeModel<TransformHealthRuleParams> {
+  return {
+    id: TRANSFORM_RULE_TYPE.TRANSFORM_HEALTH,
+    description: i18n.translate('xpack.transform.alertingRuleTypes.transformHealth.description', {
+      defaultMessage: 'Alert when transform jobs experience operational issues.',
```

Comment:
Remove the word `jobs` here as we don't normally use that term for transforms.

Reply:
Fixed in https://github.com/elastic/kibana/pull/112277/commits/25e205ddab68572209b771a93a6d4ffee602b054